core/src/main/java/org/apache/carbondata/core/metadata/AbsoluteTableIdentifier.java

```diff
@@ -99,8 +98,9 @@ public static AbsoluteTableIdentifier fromTablePath(String tablePath) {
   }
 
   public String getTablePath() {
-    return getStorePath() + File.separator + getCarbonTableIdentifier().getDatabaseName() +
-        File.separator + getCarbonTableIdentifier().getTableName();
+    return getStorePath() + CarbonCommonConstants.FILE_SEPARATOR + getCarbonTableIdentifier()
```

Comment:
I have modified in renameBadRecord method also to use CarbonCommonConstants.FILE_SEPARATOR in place of File.separator